src/it/override-war-with-snapshot/verify.groovy

```diff
@@ -0,0 +1,10 @@
+import java.io.File
+import java.util.jar.JarFile
+
+assert new File(basedir, 'war-with-plugin/target/override-war-with-snapshot.war').exists()
+
+jf = new JarFile(new File(basedir, 'war-with-plugin/target/override-war-with-snapshot.war'));
+
+assert jf.getEntry('WEB-INF/plugins/override-war-with-snapshot.hpi') != null
+
+return true;
```

Comment:
we do not need to perform any checking of the last invocation of the dependant plugin as the junit test will fail if the dependency is not updated.